nomt/src/beatree/ops/bit_ops.rs

```diff
@@ -456,4 +466,23 @@ mod tests {
             assert_eq!(expected_res, res);
         }
     }
+
+    #[test]
+    fn test_separator_len() {
```

Comment:
style: here and elsewhere, naming tests `test_` is redundant

Reply:
solved in #466